orchagent/routeorch.cpp

```diff
@@ -490,14 +490,20 @@ bool RouteOrch::removeNextHopGroup(IpAddresses ipAddresses)
 
         m_syncdNextHopGroups.erase(ipAddresses);
     }
-
     return true;
 }
 
 void RouteOrch::addTempRoute(IpPrefix ipPrefix, IpAddresses nextHops)
 {
     SWSS_LOG_ENTER();
 
+    /* Check if the same temporary route is already inserted or not */
```

Comment:
need add more comments

---
In reply to: [119508123](https://github.com/Azure/sonic-swss/pull/230#discussion_r119508123) [](ancestors = 119508123)